lib/src/command/lish.dart

```diff
@@ -381,3 +477,26 @@ String _readableFileSize(int size) {
     return '<1 KB';
   }
 }
+
+class _Publication {
```

Comment:
I personally love tuples, but feel free to use a class. I just couldn't help typing it up:
```dart
typedef _Publication = ({
  Uint8List packageBytes,
  int warningCount,
  int hintCount,
  Pubspec pubspec,
});

extension on _Publication {
  String get warningCountMessage {
    return [
      '\n',
      'Package has $warningCount ${pluralize('warning', warningCount)}',
      if (hintsCount > 0)
        ' and $hintCount ${pluralize('hint', hintCount)}',
      '.',
    ].join('');
  }
}
```